src/form-builder/CheckboxField.tsx

```diff
@@ -34,6 +34,11 @@ const CheckboxField = (props: CheckboxProps): JSX.Element => {
   );
   const id = props.id || props.name;
 
+  const onChange = (e: Event) => {
+    field.onChange(e);
+    if (props.onValueChange) props.onValueChange(e);
```

Comment:
Does the field.onChange also have to happen if we pass a custom onValueChange?

Reply:
I believe that `field.onChange` will return `handleChange` function from formik which will update the field in the state.